arches/app/templates/views/components/search/search-results.htm

```diff
@@ -8,6 +8,23 @@
 
     <div class="search-listing" role="listitem" data-bind="event: { mouseover: mouseoverInstance, mouseout: mouseoverInstance('')}, css: {'selected': selected()}">
         <div style="display: flex">
+            <div style="display: flex;">
+                <div 
+                    style="display: flex; align-items: center; margin: 10px; background-color: lightgrey;" 
+                    data-bind="
+                        visible: thumbnail,
+                        event: { 
+                            mouseover: mouseoverThumbnail,
+                            mouseout: mouseoutThumbnail,
+                        }
+                    "
+                >
+                    <img style="max-width: 70px;" data-bind="attr:{src: thumbnail}">
+                </div>
+
+                <img style="position: absolute; left: 380px; display: none;" data-bind="attr:{src: thumbnail}">
```

Comment:
Absolute positioning with left/right is a problem for RTL. Is there another way to do this?

Reply:
fixed in latest